public/app/plugins/datasource/testdata/components/SimulationQueryEditor.tsx

```diff
@@ -139,18 +133,13 @@ export const SimulationQueryEditor = ({ onChange, query, ds }: EditorProps) => {
           <Input type="text" placeholder="optional" value={simQuery.key.uid} onChange={onUIDChanged} />
         </InlineField>
       </InlineFieldRow>
-      <div>
-        <Form onSubmit={onSubmitChange}>
-          {({ register }) => (
-            <FieldSet>
-              <TextArea {...register('config')} defaultValue={JSON.stringify(config.value, null, 2)} rows={7} />
-              <Button type="submit">Submit</Button>
-            </FieldSet>
-          )}
-        </Form>
-        SCHEMA:
-        <pre>{JSON.stringify(current.details?.config.schema, null, 2)}</pre>
-      </div>
+      <SimulationSchemaForm
+        onChange={onSchemaFormChange}
+        config={cfgValue ?? config.value}
+        schema={current.details?.config.schema ?? { fields: [] }}
+      />
+      SCHEMA:
```

Comment:
we can hide this now...

Reply:
Do we want a switch to show the schema on demand or just not show it at all?